third_party/terraform/website/docs/d/app_engine_default_service_account.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+subcategory: "App Engine"
+layout: "google"
+page_title: "Google: google_app_engine_default_service_account"
+sidebar_current: "docs-google-datasource-app_engine-default-service-account"
+description: |-
+  Retrieve the default App Engine service account used in this project
+---
+
+# google\_app_engine\_default\_service\_account
+
+Use this data source to retrieve the default app engine service account for the specified project.
+
+## Example Usage
+
+```hcl
+data "google_app_engine_default_service_account" "default" {
+}
+
+output "default_account" {
+  value = data.google_app_engine_default_service_account.default.email
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project` - (Optional) The project ID. If it is not provided, the provider project is used.
+
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `email` - Email address of the default service account used by VMs running in this project
```

Comment:
This should probably be used by App Engine or something similar, right?

Reply:
yeah, App Engine and Cloud Functions default to this service account.